client/blocks/disconnect-jetpack/index.jsx

```diff
@@ -10,7 +10,8 @@ import Gridicon from 'gridicons';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 import { noop } from 'lodash';
-import page from 'page';
+import { navigate } from 'state/ui/actions';
+import { withAnalytics } from 'state/analytics/actions';
```

Comment:
Should be internal imports (further down) where we're already importing other symbols from both `state/ui/actions` and `state/analytics/actions` 😁

Reply:
Good catch, it's fixed now, thanks!